docs/platform-support.rst

```diff
@@ -164,13 +164,13 @@ NVIDIA Network Operator has been validated in the following scenarios:
      - 
    * - Red Hat Core OS
      - 
-     - 4.12-4.15
+     - 4.12-4.16
```

Comment:
4.14-4.16

This is what we use in the OCP bundle:
https://github.com/Mellanox/network-operator/blob/master/Makefile#L50

Reply:
supporting 3 releases

Reply:
@noama-nv so, what should we have here?

Reply:
aligned with Makefile